packages/flutter/lib/src/services/flutter_version.dart

```diff
@@ -0,0 +1,50 @@
+// Copyright 2014 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+// @docImport 'dart:io'
+
+/// [FlutterVersion] contains various meta information about with which
+/// Flutter version this app was compiled.
+/// It tries to mimic the information that is seen when executing `flutter --version`.
+///
```

Comment:
I think this can be tightened up and better match Flutter's documentation style:

```suggestion
/// Details about the Flutter version this app was compiled with,
/// corresponding to the output of `flutter --version`.
///
```